build/stf-run-ci/tasks/preflight_checks.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: Wait for Service Telemetry Operator to be Succeeded
+  shell: |
+    while ! oc get csv | grep service-telemetry-operator | grep Succeeded; do echo "waiting for Service Telemetry Operator..."; sleep 3; done
```

Comment:
Is this subject to any built-in timeout when running locally, or just the global one from the CI system?

Reply:
Just the global one from the CI system.

Reply:
Okay but the first time it annoys me I'm adding an extra timeout